### ISSUE TITLE
include xml in xml

### ISSUE THREAD

**Issue by GoogleCodeExporter** [rohitrng/vqmod#90]
```
i want to make a patch like following structure with vqmod. the main thing im 
using is including xml in another xml but not sure if vxmod xml compatibility  
supports that 
the include xml tag im using is 

<xi:include href="mypatch/model.xml"/>

my patcx
1.mypatch.xml  (include following patches in this xml)
2.mypatch/model.xml
3.mypatch/view.xml
3.mypatch/controller.xml

What is the expected output? What do you see instead?
si i expecting all xmls getting merge in main one but its not working. if vqmod 
supports this one example could be very helpful.

great module by the way great job.
thanks.

vQmod Version:2
Server Operating System: xampp php 5


Please provide any additional information below.


```

Original issue reported on code.google.com by `ismailxs...@gmail.com` on 26 Mar 2013 at 4:01

**Comment by GoogleCodeExporter** [comment]
Original comment by `DJG6...@gmail.com` on 26 Mar 2013 at 4:13
- Changed state: **Invalid**

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
ok here is the FIX Tested on opencart 1.5
check this line-> /* FIX@memiiso */ $dom->xinclude();/* FIX@memiiso */

    /**
     * VQMod::_parseMods()
     *
     * @return null
     * @description Loops through xml files and attempts to load them as VQModObject's
     */
    private function _parseMods() {

        $dom = new DOMDocument('1.0', 'UTF-8');
        foreach($this->_modFileList as $modFileKey => $modFile) {
            if(file_exists($modFile)) {
                if(@$dom->load($modFile)) {
                    /* FIX@memiiso */ $dom->xinclude();/* FIX@memiiso */
                    $mod = $dom->getElementsByTagName('modification')->item(0);
                    $this->_mods[] = new VQModObject($mod, $modFile, $this);
                } else {
                    $this->log->write('DOM UNABLE TO LOAD: ' . $modFile);
                }
            } else {
                $this->log->write('FILE NOT FOUND: ' . $modFile);
            }
        }
    }
```

Original comment by `ismailxs...@gmail.com` on 26 Mar 2013 at 4:22
